squid/values.yaml

```diff
@@ -119,17 +119,13 @@ ingress:
   #      - chart-example.local
 
 resources:
-  {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  requests:
```

Comment:
Do we need this in the test env as well? What's the reason for providing default values? I wonder whether setting those might cause some hard-to-trace errors in CI at some point .

Reply:
Yes. Unless it won't work without defining it.